shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitAnalysisResult.java

```diff
@@ -60,4 +69,8 @@ public ShortCircuitAnalysisResult setNetworkMetadata(NetworkMetadata networkMeta
         return this;
     }
 
+    public List<FeederResult> getFeederResults() {
+        return feederResults;
```

Comment:
Good practice for getters of lists: never directly pass the lists to prevent them to be modified without security. Always pass a unmodifiable copy: ```Collections.unmodifiableList(feederResults)```